frontend/app/auto-imports.d.ts

```diff
@@ -526,7 +526,6 @@ declare module 'vue' {
     readonly createPinia: UnwrapRef<typeof import('pinia')['createPinia']>
     readonly createProjection: UnwrapRef<typeof import('@vueuse/math')['createProjection']>
     readonly createReactiveFn: UnwrapRef<typeof import('@vueuse/core')['createReactiveFn']>
-    readonly createRef: UnwrapRef<typeof import('@vueuse/core')['createRef']>
```

Comment:
I need to think the best way to avoid this happening, the removal here is correct, but the original addition was mistaken because this is from a newer version that was not in `bugfixes` I think since we removed almost all of our code from the auto-imports, it should be possible to add a check here, so that this happens only if `pnpm-lock.yaml` is modified